README.md

```diff
@@ -132,7 +132,11 @@ Alternatively, refer to the [developer instructions](#developer-instructions) fo
                                      caseless sub-string)
     --max-downloads NUMBER           Abort after downloading NUMBER files
     --min-filesize SIZE              Do not download any videos smaller than
-                                     SIZE (e.g. 50k or 44.6m)
+                                     SIZE (e.g. 50k or 44.6m). Use match-filter
+                                     with the duration key instead of this to
+                                     filter for long videos only as this will
+                                     often download the video track and skip
+                                     the audio if they are downloaded separately.
```

Comment:
1. This file is autogenerated and should not be touched by hand.
2. `--min-filesize` works differently. `--match-filter` is based on filesize metadata while `--min-filesize` uses the actual filesize obtained via `Content-Length` HTTP header.